GREEDY.UnitTests/OCRControllerUnitTest.cs

```diff
@@ -1,56 +1,38 @@
 ﻿using Microsoft.VisualStudio.TestTools.UnitTesting;
 using GREEDY.Models;
-using System.Drawing;
-using System;
-using System.Collections.Generic;
 using System.Linq;
-using System.Text;
-using System.Threading.Tasks;
 using System.IO;
 using GREEDY.Extensions;
 
-using Emgu;
-using Emgu.CV;
-using Emgu.CV.Structure;
-using Emgu.CV.OCR;
-
-
 namespace GREEDY.UnitTests
 {
     [TestClass]
     public class OCRControllerUnitTest
     {
-        [TestMethod]
+        /*[TestMethod]
```

Comment:
Why is this commented?

Reply:
This is our old "Unit" test which isn't actuallu Unit but an Inetgration test. We will keep the class, but will have to rename it (and maybe put it to a different project, not the unit test one?). I will fix it in the future, but I want everyone to still be able to see the code so i left it commented

Reply:
I agree to have another project for integration testing